targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFabCommon/Public/TestFramework/PlayFabTestContext.h.ejs

```diff
@@ -116,7 +115,7 @@ public:
     UFUNCTION()
     void EndTest(PlayFabApiTestFinishState InFinishState = PlayFabApiTestFinishState::PASSED, FString InMessage = "")
     {
-        UE_LOG(LogPlayFabTests, Log, TEXT("Ending Test: %s."), *testName);
+        UE_LOG(LogPlayFabCommon, Log, TEXT("Ending Test: %s."), *testName);
```

Comment:
This is slightly confusing to me.
You are adding LogPlayFabTests below, but you're removing multiple uses of it up here.
Can you clarify that?